crates/floresta-node/src/json_rpc/server.rs

```diff
@@ -232,6 +237,163 @@ impl<Blockchain: RpcChain> RpcImpl<Blockchain> {
             .await
             .map_err(|_| JsonRpcError::Node("Failed to get peer info".to_string()))
     }
+
+    async fn get_blockheader(
```

Comment:
35aa245c8bb50b6de099be547c3a8f3cb6b55abd

This function should be in `blockchain.rs`